newton/_src/solvers/implicit_mpm/solver_implicit_mpm.py

```
@@ -1097,6 +1288,107 @@ def _allocate_by_voxels(particle_q, voxel_size, padding_voxels: int = 0):
+_NULL_COLOR = 1 << 31 - 1  # color for null nodes. make sure it is sorted last
+
+
+@wp.kernel
+def compute_color_offsets(
+    max_color_count: int,
+    unique_count: wp.array(dtype=int),
+    unique_colors: wp.array(dtype=int),
+    color_counts: wp.array(dtype=int),
+    color_offsets: wp.array(dtype=int),
+):
+    current_sum = int(0)
```

Ruff (0.14.0)

Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)